third_party/openssl/CMakeLists.txt

```diff
@@ -36,7 +36,7 @@ if(ANDROID)
         PREFIX openssl
         PATCH_COMMAND patch -p 0 < ${PROJECT_SOURCE_DIR}/dockcross-android.patch
         CONFIGURE_COMMAND ${CMAKE_COMMAND} -E env ANDROID_NDK=$ENV{CROSS_ROOT} <SOURCE_DIR>/Configure --sysroot=$ENV{CROSS_ROOT}/sysroot ${OPENSSL_PLATFORM} --prefix=${CMAKE_INSTALL_PREFIX} ${OPENSSL_BUILD_TYPE} no-shared
-        BUILD_COMMAND make
+        BUILD_COMMAND make install_sw
```

Comment:
Sounds like an `INSTALL_COMMAND`?

Reply:
Seems to work though.

Reply:
Sure, we could write `./configure && make && make install` in the CONFIGURE_COMMAND, and CMake wouldn't know about it. But we would :grin:.

It's nice if the log output says "starting install step" before running the installation step, I think :innocent:

Reply:
Let me try.